src/Aspire.Dashboard/Components/Pages/Metrics.razor

```diff
@@ -51,6 +51,10 @@
             <ClearSignalsButton SelectedResource="PageViewModel.SelectedApplication"
                                 HandleClearSignal="ClearMetrics" />
 
+            <PauseResumeButton
+                @bind-IsPaused="@PauseManager.MetricsPaused"
+                Disabled="@(PageViewModel.SelectedInstrument is null)" />
```

Comment:
Same as https://github.com/dotnet/aspire/pull/8184/files#r2006616340. Should it be disabled?

Reply:
Re-enabled!